subsys/net/lib/ocpp/Kconfig

```diff
@@ -0,0 +1,76 @@
+# Copyright (c) 2024 Linumiz
+# SPDX-License-Identifier: Apache-2.0
+
+config OCPP
+	bool "Open Charge Point Protocol [EXPERIMENTAL]"
+	depends on JSONC
+	depends on WEBSOCKET_CLIENT
+	help
+	  This option enables the open charge point protocol library
+
+if OCPP
+
+module=OCPP
+module-dep=NET_LOG
+module-str=Log level for OCPP
+module-help=Enables ocpp debug messages.
+source "subsys/net/Kconfig.template.log_config.net"
+
+config OCPP_INT_THREAD_STACKSIZE
+	int "OCPP internal thread stacksize"
+	default 4096
+	help
+	   OCPP internal thread stacksize
+
+config OCPP_WSREADER_THREAD_STACKSIZE
+	int "OCPP websocket reader thread stacksize"
+	default 4096
+	help
+	   OCPP websocket reader thread stacksize
+
+config OCPP_RECV_BUFFER_SIZE
+	int "OCPP websocket recive buffer size"
+	default 2048
+	help
+	   OCPP websocket recive buffer size
+
+config OCPP_INTERNAL_MSGQ_CNT
+	int "OCPP internal message queue count"
+	default 10
+	help
+	   Messages communication between websocket reader and internal
+	   process thread
+
+config OCPP_MSG_JSON
+	bool "PDU message format as JSON"
+	default y
+	help
+	   If y, then the Charge Point and Central System PDU message format
+	   as JSON
+
+config OCPP_PROFILE_SMART_CHARGE
+	bool "OCPP profile smart charging"
+	help
+	  choose to y to enable smart charging profile
```

Comment:
IMO this help message (and those below) are redundant with the config description, it's self-explanatory that setting config to y enables it. Either drop those help sections, or elaborate more about particular configs.